config/yurt-app-manager/manager/manager.yaml

```diff
@@ -1,9 +1,3 @@
-apiVersion: v1
-kind: Namespace
-metadata:
-  labels:
-    control-plane: yurt-app-manager
-  name: system
```

Comment:
system namespace is deleted, but manager deployment is still using system namespace, so deployment may cause  error.

Reply:
all the yaml config of yurt-app-manager is created by `kustomize`, Finally, the `kube-system` namespace will be used uniformly。